packages/account/src/Components/verification-status/verification-status.tsx

```diff
@@ -0,0 +1,28 @@
+import { Icon, Text } from '@deriv/components';
+import React from 'react';
+import styles from './verifications-status.module.scss';
+
+type TVerificationStatus = {
+    children?: React.ReactElement | null;
+    icon: string;
+    status_description?: React.ReactElement | null;
+    status_title: React.ReactElement | null;
+};
+
+const VerificationStatus = ({ children, icon, status_description, status_title }: TVerificationStatus) => {
+    return (
+        <div className={styles.container}>
+            <Icon icon={icon} size={128} />
+            <Text as='div' color='general' weight='bold' size='s' align='center' className={styles.statusTitle}>
```

Comment:
```suggestion
            <Text as='div' color='general' weight='bold' align='center' className={styles.statusTitle}>
```

since `s` is default, we don't have to declare it here